infrastructure/task/scripts/redeploy-failed-deployments.sh

```diff
@@ -50,6 +53,9 @@ for deployment in $FAILED_DEVELOPMENT_DEPLOYMENTS; do
   if [[ $deployment =~ "dpl-cms" ]]; then
     continue
   fi
+  if [[ $deployment =~ "dpl-bnf" ]]; then
+    continue
+  fi
```

Comment:
These changes are also not related to tihs PR, right?

Reply:
no not exactly